folksonomy/api.py

```diff
@@ -195,6 +197,19 @@ async def authentication(response: Response, session: Optional[str] = Cookie(Non
         status_code=500, detail="Server error")
 
 
+def property_where(owner, k, v):
+    conditions = ['owner=%s']
+    params = [owner]
+    if k != '':
+        conditions.append('k=%s')
+        params.append(k)
+        if v != '':
+            conditions.append('v=%s')
+            params.append(v)
+    where = " AND ".join(conditions)
+    return where, params
+
+
 @app.get("/products/stats", response_model=List[ProductStats])
 async def product_stats(response: Response,
```

Comment:
It can be done in another PR, but it would be nice to have black formatting

Reply:
I agree but it's better to have a single commit of reformatting.